docs/rules/no-restricted-jest-methods.md

```diff
@@ -0,0 +1,55 @@
+# Disallow specific `jest.` methods (`no-restricted-jest-methods`)
+
+💼 This rule is enabled in the following
+[configs](https://github.com/jest-community/eslint-plugin-jest/blob/main/README.md#shareable-configurations):
+`all`.
+
+<!-- end rule header -->
+
+You may wish to restrict the use of specific `jest` methods.
+
+## Rule details
+
+This rule checks for the usage of specific methods on the `jest` object, which
+can be used to disallow curtain patterns such as spies and mocks.
```

Comment:
just for my non native speaker understanding - are we really talking about `curtain patterns` here or is this a typo of `certain patterns`?

Reply:
typo 😀 

wanna send a PR fixing it?

Reply:
More than happy to, coming right up

Reply:
https://github.com/jest-community/eslint-plugin-jest/pull/1259